assets/css/styles.scss

```diff
@@ -210,6 +210,27 @@ div .highlight {
   margin-top: 0;
 }
 
+.inline {
+  display: inline-flex;
+	@include border-radius(0.4em);
```

Comment:
This indentation seems pretty weird?

Reply:
![image](https://user-images.githubusercontent.com/10703445/94565959-731c3100-0287-11eb-9046-656e7ab4cd86.png)

Wonder why, looks fine locally.

Reply:
Will just reformat it again.

Reply:
![image](https://user-images.githubusercontent.com/10703445/94566194-b37baf00-0287-11eb-92c5-fb5a40fb4960.png)

 Haha, tabs vs spaces 😂

Reply:
Fixed.